openmrs/i18n/clinical/locale_en.json

```diff
@@ -5,7 +5,7 @@
   "MODULE_LABEL_ACTIVE_PATIENTS_BY_LOCATION": "PatientsByLocation",
   "MODULE_LABEL_OPD_1_KEY": "OPD-1",
   "MODULE_LABEL_ALL_KEY": "All",
-
+  "MODULE_LABEL_NOTIFICATIONS_KEY": "Notifications",
```

Comment:
i think this should be added to other locales without which it fails on a different locale
<img width="283" alt="image" src="https://github.com/user-attachments/assets/17e96358-09d0-4a5c-8a96-a7c4d084a282">